docs/user/reporting/reporting-csv-limitations.asciidoc

```diff
@@ -0,0 +1,10 @@
+We recommend using CSV reports to export moderate amounts of data only. The feature enables analysis of data in external tools, but it is not intended for bulk export or to backup Elasticsearch data. Report timeout and incomplete data issues are likely if you are exporting data where:
+
+- More than 250 MB of data is being exported
+- Data is stored on slow storage tiers
+- Any shard needed for the search is unavailable
+- Network latency between nodes is high
+- Cross-cluster search is used
+- ES|QL is used and result row count exceeds the limits of ES|QL queries
```

Comment:
ES|QL wasn't available in 7.17, so maybe we should drop the mention from 7.17 docs?